src/azure-cli/requirements.py3.Darwin.txt

```diff
@@ -58,6 +58,7 @@ azure-mgmt-monitor==5.0.0
 azure-mgmt-msi==7.0.0
 azure-mgmt-netapp==10.1.0
 azure-mgmt-policyinsights==1.1.0b4
+azure-mgmt-postgresqlflexibleservers==1.0.0
```

Comment:
Will there be another PR remove `azure-mgmt-rdbms` dependency?

Reply:
Will MySql and MariaDb teams be adding change to their dependency? Left in since those commands are still dependent.

Reply:
mysql has updated dependency https://github.com/Azure/azure-cli/pull/30123

not sure if mariadb is covered